src/services/users.serveces.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { v4 } = require('uuid');
+
+let users = [];
+
+const getUsers = () => users;
+
+const createUser = (name) => {
+  const user = {
+    name,
+    id: v4(),
+  };
+
+  users.push(user);
+
+  return user;
+};
+
+const getUser = (id) => {
+  return users.find(user => user.id === id) || null;
+};
+
+const deleteUser = (id) => {
+  const newUsers = users.filter(user => user.id !== id);
+
+  users = newUsers;
+};
+
+const updateUser = ({ id, name }) => {
+  const user = getUser(id);
+
+  Object.assign(user, { name });
```

Comment:
The updateUser function directly modifies the user object. This might lead to unintended side effects. Consider creating a new object with the updated values.

Reply:
To be honest, I don't understand this comment because in this case we should to delete the real user from array and to create the new one? But if it will not lead to the same side effects? Now there aren`t mentors on the platform, so maybe you could help me)